### PR TITLE
Added some more fluff

### PR DESCRIPTION
**Pull request by CptSpaceToaster**

### CODE REVIEW

File: fluffy.py [hunk 1]
```diff
@@ -75,29 +83,70 @@ def intoduction(topic):
...
+        "a regarded efficiency expert from",
+    ))
+    yield random.choice((
+        "a competing company,",
+        "a promising startup,",
+        "a respected sales firm,",
+    ))
+    yield random.choice((
+        "A " + topic + " is a great way to synergize content.",
+        "Implementing a " + topic + " can be a quite a challending task.",
```
Comment (jacebrowning) [inline fluffy.py]: "be a quite a challending"?

File: fluffy.py [hunk 2]
```diff
@@ -52,18 +52,26 @@ def article(topic=None, company=None):
...
         "In spite of media reactions,",
+        "In this highly advanced technological age,",
+        "There will always come a time when a product becomes obsolete, but",
     ))
     yield random.choice((
-        "the medical industry",
-        "the aerospace industry",
-        "safety-critical application development",
-        "automotive infotainment",
+        "The experience carried by our high-quality workers",
```
Comment (jacebrowning) [inline fluffy.py]: Should these really be capitalized? These phrases are inserted mid-sentence.

File: fluffy.py [hunk 3]
```diff
@@ -40,7 +40,7 @@ def article(topic=None, company=None):
     ))
     company = company or "ACME"
 
-    yield from intoduction(topic)
+    yield from introduction(topic, company)
```
Comment (jacebrowning) [inline fluffy.py:47]: I get an `UnboundLocalError` if the program is called as: `$ python3 fluffy.py name`.

File: fluffy.py [hunk 4]
```diff
@@ -75,29 +83,70 @@ def intoduction(topic):
...
     yield random.choice((
-        "Ned Flanders",
-        "Lisa Simpson",
+        "Ned Flanders,",
+        "Lisa Simpson,",
+        "Mr. Bill,",
+        "Bill Nye,",
+        "Dr. Doofenshmirtz,",
+        "Columbo,",
+        "The most honorable sheriff of Nottingham,",
```
Comment (jacebrowning) [inline fluffy.py]: "The" should be lowercase here.

File: fluffy.py (hunk 3 above)
Comment (CptSpaceToaster) [inline fluffy.py:47]: I do too, but filling in "ACME" in the argument space generates the same error, Are there arg-parsing issues?

**Edit:** Yes there were.

**Pull request merged**